Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -186,7 +186,7 @@ extension KeyboardShortcuts {
 
 					NSAlert.showModal(
 						for: self.window,
-						message: "This keyboard shortcut cannot be used as it's already used by the “\(menuItem.title)” menu item."
+                        message: String(format: "This keyboard shortcut cannot be used as it's already used by the “%@” menu item.".localized, menuItem.title)
```

Comment:
Is not this aligned vertically?
![image](https://user-images.githubusercontent.com/33802186/91557484-2206dd80-e967-11ea-8830-d17398eaba5d.png)